app/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Tue Jul 02 23:05:03 AEST 2019
```

Comment:
I think you might have set up your project wrong, the gradle dir should be at the same level as the app, not inside it. Remove these files from the commit as they aren't needed (you don't have to delete them locally)